### PR TITLE
Price Quote Tool and other fixes

### PR DESCRIPTION
**Pull request by dhuangg**
fixed price quote tool, dates, and registration landing loading

### CODE REVIEW

File: src/components/FeatureViews/Accounts/TabComponents/EnrollmentView.js [hunk 1]
```diff
@@ -173,6 +160,10 @@ const CourseSessionStatus = () => {
         setActiveTab(newTab);
     };
 
+    const handleHighlightSwitch = (e) => {
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Accounts/TabComponents/EnrollmentView.js]: Remove unused parameter.

File: src/components/FeatureViews/Scheduler/SchedulerUtils.js [hunk 2]
```diff
@@ -13,6 +13,42 @@ export const BootstrapInput = withStyles(theme => ({
...
+            'BlinkMacSystemFont',
+            '"Segoe UI"',
+            'Roboto',
+            '"Helvetica Neue"',
+            'Arial',
+            'sans-serif',
+            '"Apple Color Emoji"',
+            '"Segoe UI Emoji"',
+            '"Segoe UI Symbol"',
+        ].join(','),
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Scheduler/SchedulerUtils.js]: Do we really need to specify all this? lol

File: src/components/Form/TutoringPriceQuote.js [hunk 3]
```diff
@@ -257,12 +288,16 @@ const TutoringPriceQuote = ({courseType, handleUpdatePriceFields, tuitionConfirm
...
-                                    variant={"outlined"}
-                                />
+                                    <InputLabel htmlFor={"hourly-tuition"}>Hourly Tuition</InputLabel>
+                                <div
+                                    style={{
+                                        padding: '10px 26px 10px 12px',
+                                    }}
+                                >
+                                    <Typography variant="h6">
+                                        {hourlyTuition || ''}
```
Comment (NeilAllavarpu) [inline src/components/Form/TutoringPriceQuote.js]: Unnecessary `||`, if `hourlyTuition` is undefined or null, nothing will be rendered regardless.

File: src/components/FeatureViews/Scheduler/SchedulerUtils.js (hunk 2 above)
Comment (dhuangg) [reply src/components/FeatureViews/Scheduler/SchedulerUtils.js]: this may have been copy pasta'd lol

**Pull request merged**